hlint/hlint.dhall

```diff
@@ -882,4 +882,18 @@ in [ rule.Arguments { arguments =
    , warnLifted "putBSLn" "x"
    , warnLifted "putLBS" "x"
    , warnLifted "putLBSLn" "x"
+
+   ------------
+   -- Extra
+   ------------
+
+   -- Tuple
+   , hintNote "fmap (,a) (f a)"         "traverseToFst f a" "Use `traverseToFst`"
```

Comment:
How do I make this aware of the need to `import Relude.Extra.Tuple`?

Reply:
What do you mean here? You want to tell users that `traverseToFst` comes from the `Relude.Extra.Tuple` module? In that case you can just adjust the message of the note:

```
Use `traverseToFst` from `Relude.Extra.Tuple`
```

Reply:
I mean a way to make hlint itself aware of the required import, so if someone were to run it with `--refactor`, it wouldn't break their code by adding a usage of it without the import. Is this possible? If not, then I will just add it to the message for the user.

Reply:
> Is this possible? If not, then I will just add it to the message for the user.

I think it's not possible. It's one of the reasons why it's just a hint, not a warning. So adding just message is okay 👍 

Note, that if you're using `base-noprelude` to add custom relude, you only need to add this import in one place for your application.